TESTS/mbed_hal/sleep_manager_racecondition/main.cpp

```diff
@@ -37,8 +39,8 @@ void sleep_manager_multithread_test()
 {
     {
         Callback<void()> cb(sleep_manager_locking_thread_test);
-        Thread t1;
-        Thread t2;
+        Thread t1(osPriorityNormal, TEST_STACK_SIZE);
```

Comment:
this multithreaded test should be guarded by `MBED_RTOS_SINGLE_THREAD` . I am looking at what this macro is changing, and found only tests that use it and then toolchains python files that define it. Pointers?